src/util_regression.mjs

```diff
@@ -93,7 +93,7 @@ util_regression.KalmanFilter = function(F, H, Q, R, P_initial, X_initial) {
  */
 util_regression.KalmanFilter.prototype.update = function(z) {
     // Here, we define all the different matrix operations we will need
-    var add = numeric.add, sub = numeric.sub, inv = numeric.inv, identity = numeric.identity;
+    var add = mathjs.add, sub = mathjs.subtract, inv = mathjs.inv, identity = mathjs.identity;
     var mult = mat.mult, transpose = mat.transpose;
```

Comment:
We can also use the `multiply` and `transpose` function provided by mathjs here.

Reply:
Please do. Multiplication is a big computational bottleneck and should use Math.JS's optimization implementation.

Reply:
@Skylion007 updated.